scripts-dev/release.py

```diff
@@ -444,18 +445,53 @@ def _upload() -> None:
         click.echo("Tag {tag_name} (tag.commit) is not currently checked out!")
         click.get_current_context().abort()
 
-    pypi_asset_names = [
-        f"matrix_synapse-{current_version}-py3-none-any.whl",
-        f"matrix-synapse-{current_version}.tar.gz",
-    ]
+    headers = {"Accept": "application/vnd.github+json"}
+    if gh_token is not None:
+        headers["authorization"] = f"token {gh_token}"
+
+    # First load info about the release
+    try:
+        url = (
+            f"https://api.github.com/repos/matrix-org/synapse/releases/tags/{tag_name}"
+        )
+        req = urllib.request.Request(url, headers=headers)
+        response = urllib.request.urlopen(req)
+        release_info_resp = json.loads(response.read())
```

Comment:
elsewhere we seem to use some kind of package for this (see `from github import Github`). Consistency would be nice but that's not a reason to block this

Reply:
Hmm that's a good point. And this time I don't have the excuse that it doesn't support it, because it seems like it does (for the workflow waiting, this wasn't supported).
I'll update to use it